copycat.tmux

```diff
@@ -40,7 +57,11 @@ set_start_bindings() {
 	for search_var in $stored_search_vars; do
 		key="$(get_stored_search_key "$search_var")"
 		pattern="$(get_stored_search_pattern "$search_var")"
-		tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/copycat_mode_start.sh '$pattern'"
+		if [ -z ${keytable_supported+x} ]; then
+			tmux bind-key "$key" run-shell "$CURRENT_DIR/scripts/copycat_mode_start.sh '$pattern'"
+		else
+			tmux bind-key -T "$keytable" "$key" run-shell "$CURRENT_DIR/scripts/copycat_mode_start.sh '$pattern'"
```

Comment:
Can we somehow avoid having the `tmux bind-key...` line twice?

I'm thinking something like this:

    tmux bind-key "$key_table_argument" "$key" run-shell "$CURRENT_DIR/scripts/copycat_mode_start.sh '$pattern'"

Where `$key_table_argument` would be key table argument value or a blank string when tmux doesn't have key table support.